Sources/CoreMetrics/Metrics.swift

```diff
@@ -12,6 +12,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Dispatch
```

Comment:
unnecessary, core metrics imports dispatch without need now

Reply:
@ktoso Thanks for the update & sorry for the inconvenience caused! ✌️

Reply:
Yep no worries 👍